### PR TITLE
Add GCE Eco-Devices integration

### PR DESCRIPTION
**Pull request by Aohzan**
## Proposed change

Add documentation for a new integration for GCE Eco-Devices: https://www.gce-electronics.com/fr/carte-relais-ethernet-module-rail-din/409-teleinformation-ethernet-ecodevices.html
From custom component https://github.com/Aohzan/ecodevices

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/45499
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2203
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @shawnoster, @kongo09, @matt-gnu, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16295).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by Aohzan** [comment]
> It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description.
> (message by DocsTargetBranch)

Hello, I switch the branch to current, but I did on the next branch because : 
> Added documentation for a new integration I'm adding to Home Assistant (next branch)

The link is already on the first post : 
> Link to parent pull request in the codebase: home-assistant/core#45499

File: source/_integrations/ecodevices.markdown [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+sensor:
+  - platform: ecodevices
+    host: "192.168.1.239"
+    t1_enabled: True
+    t1_name: Compteur Linky
+    c1_enabled: True
+    c1_name: Compteur panneau solaire
+    c1_icon: mdi:solar-panel
+    c1_unit_of_measurement: Wh
+    c1_device_class: power
```
Comment (frenck) [inline source/_integrations/ecodevices.markdown]: New integrations, that communicate with a device or service, should not provide an option for YAML configuration.

Please remove this functionality, as it is only intended for transitional/migration purposes of existing integrations.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/ecodevices.markdown (hunk 1 above)
Comment (Aohzan) [reply source/_integrations/ecodevices.markdown]: Ok, I will update it to delete the YAML part and provide only configuration from GUI.

For "complexe" integration (like https://github.com/Aohzan/ipx800#example, that I will subit soon too) which need a lot of parameter. The YAML isn't possible too ?

Comment (frenck) [reply source/_integrations/ecodevices.markdown]: Exceptions are possible, for example, MQTT/KNX, which define a complex protocol/specification that isn't possible with the UI at this point.

However, in this PR; that is not the case, as a config flow is provided already.

Comment (Aohzan) [reply source/_integrations/ecodevices.markdown]: yes of course, so I have to think about the ipx800v4 integration

**Comment by Aohzan** [comment]
done @frenck :)

File: source/_integrations/ecodevices.markdown [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - '@Aohzan'
+ha_config_flow: true
+ha_domain: ecodevices
+---
+
+The `ecodevices` integration allows you to get information from [GCE Eco-Devices](http://gce-electronics.com/fr/carte-relais-ethernet-module-rail-din/409-teleinformation-ethernet-ecodevices.html).
+
+## Configuration
+
+To add ecodevices to your installation, go to Configuration >> Integrations in the UI, click the button with + sign and from the list of integrations select GCE Eco-Devices.
```
Comment (klaasnicolaas) [inline source/_integrations/ecodevices.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/ecodevices.markdown (hunk 2 above)
Comment (Aohzan) [reply source/_integrations/ecodevices.markdown]: thanks

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍
⚠️ The ha_release might need an update before merging this one in.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
There are unrelated commits in this PR. Please git rebase your changes onto the latest `next` branch to clean it up. Thanks 👍

**Comment by Aohzan** [comment]
I close this PR as I messed up with by git...

**Pull request closed without merging**